examples/guess-the-sketch/src/app.py

```diff
@@ -98,27 +101,33 @@ def index():
 # The set to store the assigned players who were disconnected
 disconnected_players = set()
 sid_to_player_id = {}
+player_id_to_promptes_set = {}
 
 @socketio.on('syncSession')
 def handle_sync_session(data):
     player_id = data['playerId']
     logger.debug('Player %s: Received syncSession from sid %s', player_id, request.sid)
     if player_id in connected_players:
-        # A known player reconnected, remove it from disconnected_player set if it's in
-        disconnected_players.discard(player_id)
-        # Reset the socket id for the player
-        sid_to_player_id[request.sid] = player_id
-        join_room(player_id)
+        # A known player reconnected, i.e. refresh, for simiplicity, we just redirect to start page
+        emit('redirect', room=request.sid)
```

Comment:
Why did this change from player_id to request.sid? I thought after Zach's change we were going to use the player id instead?

Reply:
Here is the case that player reconnected, i.e. a new socket connection is established. The previous room for the same player id is linked to a different socket connection so I think room=player_id will not work? I'm not sure whether I understand how room works here correctly though.

Reply:
On line 130 room is still set to player_id - does that code branch work?

My understanding of Zach's code was that it made a map from request.sid -> player id and then used player_id since that value would be stable across reconnects. Since we aren't really trying to handle reconnects, it probably doesn't matter too much. As long as this works, I think it's something that could be fixed later.

Reply:
line 130 is the case where the connection is first establish so it works. My understanding is that in reconnection, if emit to room=player_id, it will emit event to the old socket which linked to that player id rather than the current one. Tested with room=player_id for this line and it doesn't work. So I'm going to keep it is.